packages/core/bootstrap/src/lib/metrics/index.ts

```diff
@@ -74,9 +74,30 @@ export const withMetrics =
             ? HttpRequestType.CACHE_HIT
             : HttpRequestType.DATA_PROVIDER_HIT,
       })
+
+      let sourceString
+      try {
+        if (
+          input?.data?.source &&
+          typeof input?.data?.source === 'string' &&
+          input?.data?.source !== ''
+        ) {
+          const sourceName = input?.data?.source
+          sourceString = '?SOURCE=' + sourceName.toUpperCase()
+        }
+      } catch (e: any) {
```

Comment:
What could go wrong here?

Reply:
Ah yes, I was doing a JSON.parse in there earlier, but I removed that, so try catch is no longer needed.